odoo/tools/pdf/__init__.py

```diff
@@ -217,6 +217,24 @@ def to_pdf_stream(attachment) -> io.BytesIO:
     _logger.warning("mimetype (%s) not recognized for %s", attachment.mimetype, attachment)
 
 
+def extract_page(attachment, num_page=0) -> io.BytesIO | None:
+    """Exctract a specific page form an attachement pdf"""
+    pdf_stream = to_pdf_stream(attachment)
+    if not pdf_stream:
+        return
+    try:
+        pdf = PdfFileReader(pdf_stream)
```

Comment:
Shouldn't we use OdooPdfFileReader?
I didn't know it existed before looking it up.